js/analyzer.js

```diff
@@ -67,9 +67,12 @@ Analyzer.prototype.checkConfig = function() {
  * YoastSEO.Analyzer initialization. Loads defaults and overloads custom settings.
  */
 Analyzer.prototype.init = function( args ) {
+	var metaValues = {
```

Comment:
You're right, but because there will be more arguments in the metaData object for readability it will be clearer to define them in an object. This way we can also change the metaData in a valueobject if we want to.